include/snitch/snitch_macros_test_case.hpp

```diff
@@ -11,8 +11,13 @@
         snitch::tests.add({__VA_ARGS__}, SNITCH_CURRENT_LOCATION, &ID);                            \
     void ID()
 
-#define SNITCH_TEST_CASE(...)                                                                      \
-    SNITCH_TEST_CASE_IMPL(SNITCH_MACRO_CONCAT(test_fun_, __COUNTER__), __VA_ARGS__)
+#if !(SNITCH_DISABLE)
```

Comment:
Could we use the same format here as in the other files, namely, a single big "if enabled" block, followed by a single "if disabled" block? That would make it easier to follow. I think most if not all of the `*_IMPL(...)` macros can be defined only if *snitch* is enabled, for example.